packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -5644,8 +5686,8 @@ describe("Teal Opcodes", function () {
 				assert.deepEqual(
 					stack.pop(),
 					new Uint8Array([
-						227, 23, 18, 22, 19, 23, 18, 22, 19, 23, 18, 22, 19, 23, 18, 22, 19, 23, 18, 22, 19,
-						23, 18, 22, 19, 23, 18, 22, 19, 23, 18, 28,
+						227, 23, 18, 22, 19, 23, 18, 22, 19, 23, 18, 22, 19, 23, 18, 22,
+						19, 23, 18, 22, 19, 23, 18, 22, 19, 23, 18, 22, 19, 23, 18, 28,
```

Comment:
what are these uint8Array values?

Reply:
these are expected values from `ByteBitwiseXOR` opcode. I only changed the formatting here since the lines were too long